target/linux/ramips/image/mt7620.mk

```diff
@@ -362,6 +362,20 @@ define Device/elecom_wrh-300cr
 endef
 TARGET_DEVICES += elecom_wrh-300cr
 
+define Device/engenius_esr600
+  MTK_SOC := mt7620a
+  BLOCKSIZE := 64k
+  IMAGE_SIZE := 15616k
+  IMAGES += factory.dlf
+  IMAGE/factory.dlf := $$(sysupgrade_bin) | check-size $$$$(IMAGE_SIZE) | \
+       senao-header -r 0x101 -p 0x57 -t 2
+  SUPPORTED_DEVICES += esr600
```

Comment:
You should not need this when you are adding a new device.

Reply:
Agreed.  Pushed commit to remove manipulation of SUPPORTED_DEVICES.